specification/vmware/resource-manager/Microsoft.AVS/preview/2020-07-17-preview/vmware.json

```diff
@@ -0,0 +1,3921 @@
+        "nsxtManager": {
+          "description": "Endpoint for the NSX-T Data Center manager",
+          "type": "string",
+          "readOnly": true
+        },
+        "vcsa": {
+          "description": "Endpoint for Virtual Center Server Appliance",
+          "type": "string",
+          "readOnly": true
+        },
+        "hcxCloudManager": {
+          "description": "Endpoint for the HCX Cloud Manager",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "IdentitySource": {
+      "description": "vCenter Single Sign On Identity Source",
+      "properties": {
+        "name": {
+          "description": "The name of the identity source",
+          "type": "string"
+        },
+        "alias": {
+          "description": "The domain's NetBIOS name",
+          "type": "string"
+        },
+        "domain": {
+          "description": "The domain's dns name",
+          "type": "string"
+        },
+        "baseUserDN": {
+          "description": "The base distinguished name for users",
+          "type": "string"
+        },
+        "baseGroupDN": {
+          "description": "The base distinguished name for groups",
+          "type": "string"
+        },
+        "primaryServer": {
+          "description": "Primary server URL",
+          "type": "string"
+        },
+        "secondaryServer": {
+          "description": "Secondary server URL",
+          "type": "string"
+        },
+        "ssl": {
+          "description": "Protect LDAP communication using SSL certificate (LDAPS)",
+          "type": "string",
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "x-ms-enum": {
+            "name": "SslEnum",
+            "modelAsString": true
+          }
+        },
+        "username": {
+          "description": "The ID of an Active Directory user with a minimum of read-only access to Base DN for users and group",
+          "type": "string"
+        },
+        "password": {
+          "description": "The password of the Active Directory user with a minimum of read-only access to Base DN for users and groups.",
+          "type": "string",
+          "x-ms-secret": true
+        }
+      }
+    },
+    "PrivateCloud": {
+      "description": "A private cloud resource",
+      "required": [
+        "location",
+        "sku"
+      ],
+      "allOf": [
+        {
+          "$ref": "#/definitions/TrackedResource"
+        }
+      ],
+      "properties": {
+        "sku": {
+          "description": "The private cloud SKU",
+          "$ref": "#/definitions/Sku"
+        },
+        "properties": {
+          "description": "The properties of a private cloud resource",
+          "$ref": "#/definitions/PrivateCloudProperties",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "PrivateCloudUpdate": {
+      "description": "An update to a private cloud resource",
+      "properties": {
+        "properties": {
```

Comment:
This PATCH definition needs to allow updating tags... it used to in the old API

Reply:
The service supports it. It is in the examples, but not in the 2020-03-20 API. I think it was missed when I created the PrivateCloudUpdate definition. I added it here in 592b864.